### PR TITLE
[xray] Fix broken customVolumeMounts

### PR DESCRIPTION
**Pull request by chukka**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:
[xray] Fix broken customVolumeMounts
- Added customVolumeMounts under `router`

### CODE REVIEW

**Comment by eldada** [comment]
Can we add an example in the README on use cases for this functionality? For example for custom CA certificates...

File: stable/xray/templates/xray-statefulset.yaml [hunk 1]
```diff
@@ -481,6 +500,9 @@ spec:
         volumeMounts:
         - name: data-volume
           mountPath: "{{ .Values.xray.persistence.mountPath }}"
+{{- if .Values.common.customVolumeMounts }}
```
Comment (danielezer) [inline stable/xray/templates/xray-statefulset.yaml:503]: Please use a with statement for these blocks, it is much cleaner than repeating the value twice, plus it removes the need for the if statement

Comment (chukka) [reply stable/xray/templates/xray-statefulset.yaml]: Done

**Comment by chukka** [comment]
@evannorstrand @erick-duran Please review this PR :)

**Comment by evannorstrand** [comment]
I see no issues! Will work like it did previously which is perfect!

**Pull request merged**